moderator.lua

```diff
@@ -96,12 +96,12 @@ function HandleLightningCommand(Split, Player)
 		local LookPos = GetPlayerLookPos(Player)
 		if Split[2] == nil then
 			if LookPos == nil then
-				Player:GetWorld():CastThunderbolt(Player:GetPosX(), Player:GetPosY(), Player:GetPosZ())
+				Player:GetWorld():CastThunderbolt(Vector3i(Player:GetPosX(), Player:GetPosY(), Player:GetPosZ()))
```

Comment:
cPlayer has a 'GetPosition' method.

Reply:
Player:GetPosition() uses Vector3d, but CastThunderbolt requires Vector3i.

Reply:
Yes, you could do ‘Vector3i(Player:GetPosition())’ , right?

Reply:
Good point.